pkg/util/util.go

```diff
@@ -107,7 +107,8 @@ func equalIgnorePlural(str1, str2 string) bool {
 }
 
 func IsResourceInBackup(resourceKind string, backup *velerov1.Backup) bool {
-	return IsResourceIncluded(resourceKind, backup) && !IsResourceExcluded(resourceKind, backup)
+	gr := schema.ParseGroupResource(resourceKind)
```

Comment:
I might be missing something but I don't see how putting this here solves what you meant.
From what I see right now you send to the function `IsResourceInBackup` "virtualmachines.kubevirt.io" or "virtualmachines", then  ParseGroupResource on that, and pass gr.Resource to IsResourceIncluded/excluded which gr.Resource is "virtualmachines" in either case so basically it didnt change anything?
Isnt what you need is to parse `res` from: "for _, res := range backup.Spec.ExcludedResources " in IsResourceIncluded/excluded so that if the user bring either  "virtualmachines.kubevirt.io" or "virtualmachines" it will turn to "virtualmachines"?
Also missing UT